### DISCUSSION TITLE
Use different version of gosec

### DISCUSSION THREAD

**Discussion by ab-passos** [golangci/golangci-lint#2777] [Q&A] [answered]
Hi all, 

I see that golangci-lint is using an old version of gosec. Is there a way to tune the version of a certain linter? 
Thanks in advance

**Comment by SVilgelm** [comment, marked as answer]
The golangci-linter uses `v2.11.0`: https://github.com/golangci/golangci-lint/blob/master/go.mod#L74
The latest release is `v2.11.0`: https://github.com/securego/gosec/releases

so where did you find the outdated version?
